packages/user_repository/lib/src/user_repository.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:async';
+
+import 'package:user_repository/src/domain/user.dart';
+import 'package:uuid/uuid.dart';
+
+class UserRepository {
```

Comment:
maybe you could define the abstract repository that defines all the methods that's will be or at least that will be great for user repository? s

Reply:
I will do that when implementing an injection mechanism

Reply:
Alright